developer_manual/digging_deeper/profiler.rst

```diff
@@ -0,0 +1,150 @@
+Profiler
+========
+
+Nextcloud offers a built-in profiler that can help you identifying performance issues
+with your Nextcloud application. This feature is enabled starting from Nextcloud 24.
+
+Get the app
+-----------
+
+The profiler application is available on `GitHub <https://github.com/nextcloud/profiler>`_
+you need to clone the stableX branch if you are using Nextcloud X or master if you are
+using the latest development version of Nextcloud.
```

Comment:
Why not release it to the appstore? Are other applications recommending this for installation?

Reply:
no because I don't want users to use it on production without careful consideration